docs/changelog/biweekly-report/2022-12-16-biweekly-report.md

```diff
@@ -0,0 +1,110 @@
+
+Join us at [github.com](https://github.com/GreptimeTeam/greptimedb)
+
+## Contributors list: (In alphabetical order)
+
+[@azhsmesos](https://github.com/azhsmesos) [@DiamondMofeng](https://github.com/DiamondMofeng) [@dongxuwang](https://github.com/dongxuwang)
+
+[@e1ijah1](https://github.com/e1ijah1) [@francis-du](https://github.com/francis-du) [@lbt05](https://github.com/lbt05)
+
+[@lizhemingi](https://github.com/lizhemingi) [@morigs](https://github.com/morigs) [@SSebo](https://github.com/SSebo) [@Xuanwo](https://github.com/Xuanwo)
+
+A big THANK YOU for the generous and brilliant contributions! It is people like you who are making GreptimeDB a great product. Let's keep up the good work!
+
+Join us at [github.com](https://github.com/GreptimeTeam/greptimedb)
+
+## Good first issue
+
+### Issue #667 (Closed)
+
+Link：https://github.com/GreptimeTeam/greptimedb/issues/667
+
+Issue description:
+
+When we define `TIME INDEX` column,
+for example `CREATE TABLE` SQL:
+
+```SQL
+CREATE TABLE system_metrics (
+    host STRING,
+    idc STRING,
+    ts TIMESTAMP NOT NULL,
+    PRIMARY KEY(host, idc),
+    TIME INDEX(ts)
+);
+```
+
+there are two special column specifications: `PRIMARY KEY` and `TIME INDEX`. The first one is used to define primary keys (or tags in some systems), and the second is to define which column is the special timestamp column.
+
+For `TIMESTAMP` column, a simplification is to change the grammar to:
+
+```SQL
+ts TIMESTAMP TIME INDEX,
+```
+
+By doing so, it
+
+- need not to write `NOT NULL`, this is implied by `TIME INDEX` keywords
+- need not to specify `TIME INDEX` in another line
+
+GreptimeDB now supports both of these two grammar rules. Users can choose one of them according to their preference.
+
+### Issue #602 (**Help Wanted**)
+
+Link:  https://github.com/GreptimeTeam/greptimedb/issues/602
+
+Issue description:
+
+Add system tables for inner metrics.
+It's good to have a metrics table like the one in [Databend](https://databend.rs/doc/sql-reference/system-tables/system-metrics) or [ClickHouse](https://clickhouse.com/docs/en/operations/system-tables/metrics/). Users can easily "select" on the metrics table to see DB's important running status. With the metrics table, we can integrate other visualization tools or dashboards that support SQL like Grafana, to export some meaningful time series data.
+1. In datanode, create a new table called `metrics`, with a schema including at least `metric`, `value`, and `description`, and also feel free to add other columns when necessary. Register it into the default catalog as our `ScriptsTable` does.
+2. Implement "select" for the metrics table. Instead of actually storing the metrics, call the `render()` method of `PROMETHEUS_HANDLE`, and parse the result (Prometheus lines) as table output.
+3. In frontend, create the same metrics table as well. Unlike `DistTable`, the metrics table does not do distributed queries.
```

Comment:
Unlike `DistTable`, the metrics table does not do distributed queries.
```suggest to change "do" distributed queries to other verbs, such as, does not execute

Reply:
@apdong2022 it seems the suggestions are lost.

Reply:
> @apdong2022 it seems the suggestions are lost.

please check again